tests/ui/js/base-page-storyEditTests.js

```diff
@@ -863,6 +879,303 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "An actual heading",
+                alternativeText: "",
+                description: "",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "Some alternative text",
+                description: "",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "A real description",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "",
+                transcript: "A real transcript",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "",
+                transcript: "",
+                mediaUrl: "Not really a URL"
+            },                  blockContentValues: "default",  expectedEmpty: false
+        }
+    ];
+
+    sjrk.storyTelling.base.page.storyEditTester.verifyIsEmptyBlock = function (blockContentValues) {
+        fluid.each(isEmptyBlockTestCases, function (testCase, index) {
+            var blockContentValuesToUse = testCase.blockContentValues === "default" ? blockContentValues : testCase.blockContentValues;
+
+            var actuallyEmpty = sjrk.storyTelling.base.page.storyEdit.isEmptyBlock(testCase.block, blockContentValuesToUse[testCase.block.blockType]);
+            jqUnit.assertEquals("Block emptiness state for test case " + index + " is as expected", testCase.expectedEmpty, actuallyEmpty);
```

Comment:
May want to consider naming the test cases, or providing some more detail in the output string. If one were to break, it would be hard to locate it because of the number test cases provided.